harness/determined/deploy/local/cluster_utils.py

```diff
@@ -15,6 +16,9 @@
 from determined.deploy.errors import MasterTimeoutExpired
 from determined.deploy.healthcheck import wait_for_master
 
+AGENT_NAME_DEFAULT = "det-agent-<hostname>"
```

Comment:
I think this will save a lot of headache, and I can't think of a reason not to do it

```suggestion
AGENT_NAME_DEFAULT = "det-agent-{socket.gethostname()}"
```